commands/show-bl-ip.go

```diff
@@ -130,21 +120,29 @@ func printBLDestIPs(c *cli.Context) error {
 	}
 
 	if human {
-		err = showBLIPsHuman(data, connected, false)
+		err = showBLIPsHuman(data, connected, showNetNames, false)
 		if err != nil {
 			return cli.NewExitError(err.Error(), -1)
 		}
 	} else {
-		err = showBLIPs(data, connected, false, c.String("delimiter"))
+		err = showBLIPs(data, connected, showNetNames, false, c.String("delimiter"))
 		if err != nil {
 			return cli.NewExitError(err.Error(), -1)
 		}
 	}
 	return nil
 }
 
-func showBLIPs(ips []host.AnalysisView, connectedHosts, source bool, delim string) error {
-	headers := []string{"IP", "Connections", "Unique Connections", "Total Bytes"}
+func showBLIPs(ips []blacklist.IPResult, connectedHosts, showNetNames, source bool, delim string) error {
+	var headers []string
```

Comment:
Consider assigning the values to the headers variable all in one shot rather than assigning some of the values and then appending on the rest in a second statement. For instance, look at line 96 of the showConns function in show-long.connections.go file. The former method has the advantage that most of the header values only need to be modified in a single place and might reduce some trouble-shooting headaches later, the latter method is a bit faster as the program doesn't have to allocate space for a new array and then copy the elements due to the append function.  I saw the mismatch in logic-approach in a few locations.

Reply:
Changed, although it turned into a 6-way if statement.